.github/ISSUE_TEMPLATE/dump_request.yml

```diff
@@ -0,0 +1,37 @@
+name: Global Observation Dump Request
+description: Request additional dates be added to a machine's global dump archive
+labels: ["Static Data Mgmt"]
+assignees:
+  - KateFriedman-NOAA
+  - WalterKolczynski-NOAA
+
+body:
+  - type: dropdown
+    attributes:
+      label: Machine
+      options:
+        - WCOSS2
+        - Hera/Ursa
+        - Orion/Hercules
+        - Jet
+      multiple: true
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Start date
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: End date
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Additional information
+      placeholder: |
+        Any additional information needed (experimental obs, etc.)
```

Comment:
Can you add a pickup location input?

Also, I'd love to also use this issue template for requests for experimental data pre-ops that I pick up on WCOSS2 (from obsproc usually). I think the existing fields you included plus a pickup location field would cover that.

Reply:
See above.